magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/toolbar/ToolbarHelper.kt

```diff
@@ -25,16 +23,6 @@ object ToolbarHelper : LifecycleAwareComponent() {
     }
   }
 
-  override fun onStart(context: Context) {
-    shownScope.launch {
-      NavigationPropagator.events
-        .filterIsInstance<NavigationLifecycleEvent.NavigatedFrom>()
-        .collect { event ->
-          toolbarView?.reset()
-        }
-    }
```

Comment:
We should replace this with a call to `NavigationPropagator.addListener(...)` (or whatever the API is) and removing it in `onStop`